templates/init_runner_script.sh

```diff
@@ -81,15 +81,10 @@ EOF'
 
 sudo subscription-manager repos --enable=rhel-7-server-extras-rpms --enable=rhel-7-server-optional-rpms
 sudo yum install -y docker
-elif [ "${LINUX_OS_VERSION}" = "8" ]
+elif [ "${LINUX_OS_VERSION}" = "8" || "${LINUX_OS_VERSION}" = "9" ]
 then
-sudo dnf install -y podman-docker
+sudo dnf install -y podman-docker podman
```

Comment:
Optional: Might not be necessary as the package could be implicitly included, but doesn't hurt to explicitly mention it as well. 
```suggestion
sudo dnf install -y podman-docker
```